pkg/controller/plan/adapter/vsphere/client.go

```diff
@@ -318,6 +324,58 @@ func (r *Client) getClient(vm *model.VM, hosts util.HostsFunc) (client *vim25.Cl
 	return
 }
 
+// CipherSuiteId copied and edited the CipherSuiteName from tls lib
+func CipherSuiteId(name string) uint16 {
```

Comment:
I don't like the code duplicity,
@mansam do you know some nice place where it would be nice fit for this?